.github/scripts/transform_dbt_test_results.py

```diff
@@ -86,6 +87,7 @@ def __init__(
         self,
         name: str,
         table_name: str,
+        column_name: typing.Optional[str],
```

Comment:
Note that the `column_name` attribute of a dbt manifest node can be null in cases where the test was defined on a model rather than on a column (e.g. table uniqueness tests).